docs/networks.md

```diff
@@ -3,14 +3,13 @@
 
 Networks in the HaaS have three fields:
 
-- 'creator' represents what project created it; or 'admin' if it was
-  administrator-created.  If there are still networks created by a project,
-  then that project cannot be deleted.  Only the creator has the ability to
+- 'owner' represents what project owns it; or 'admin' if it was
+  administrator-created.  If there are still networks owned by a project,
+  then that project cannot be deleted.  Only the owner has the ability to
   delete a network.
 
-- 'access' represents which project's nodes can access it, or '' (the
-  empty string) if all   projects' nodes can.  This will eventually be extended
-  into a ACL-style approach.
+- 'access' represents which projects' nodes can access it, or [] (the
```

Comment:
`[]` isn't rendering here. You can check it either by playing around with one of these comments boxes and hitting "Preview", or by clicking the document button in the diffs.

Maybe try: 
```markdown
`[]`
```